docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,99 @@
+"""
+Coriolis parameters over the UK
+===============================
+
+This example shows the coriolis parameters for the area encompassing the UK.
```

Comment:
Coriolis is a proper noun so should be capitalised. A more correct term might be the "Coriolis frequency".